image/descriptor.go

```diff
@@ -73,7 +73,17 @@ func findDescriptor(w walker, name string) (*descriptor, error) {
 	}
 }
 
-func (d *descriptor) validate(w walker) error {
+func (d *descriptor) validate(w walker, mts []string) error {
```

Comment:
On Wed, Sep 07, 2016 at 12:00:38AM -0700, Antonio Murdaca wrote:
> > @@ -73,7 +73,17 @@ func findDescriptor(w walker, name string) (*descriptor, error) {
> >  	}
> >  }
> >  
> > -func (d *descriptor) validate(w walker) error {
> > +func (d *descriptor) validate(w walker, mts []string) error {
> 
> Isn't it better to open another issue tracking this? 

You're adding ‘mts’ here, and I don't think we want it ;).  If folks
feel like the rest of this PR is a step forward, I'm ok with this
landing and me filing a subsequent PR to roll this part back out in
favor of --unpackable.